Fix github workflow build

Following "Build the extension" step is failing in github workflow. 
```
python -m jupyterlab.browser_check
```

```
Starting Chrome Headless
Navigating to page: file:///home/runner/.local/share/jupyter/runtime/jpserver-2551-open.html
Waiting for page to load...
/opt/hostedtoolcache/Python/3.7.10/x64/share/jupyter/lab/browser_test/chrome-test.js:55
  throw up;
  ^

TimeoutError: waiting for selector "#browserTest" failed: timeout 100000ms exceeded
    at new WaitTask (/opt/hostedtoolcache/Python/3.7.10/x64/share/jupyter/lab/browser_test/node_modules/puppeteer/lib/DOMWorld.js:407:34)
    at DOMWorld._waitForSelectorOrXPath (/opt/hostedtoolcache/Python/3.7.10/x64/share/jupyter/lab/browser_test/node_modules/puppeteer/lib/DOMWorld.js:332:26)
    at DOMWorld.waitForSelector (/opt/hostedtoolcache/Python/3.7.10/x64/share/jupyter/lab/browser_test/node_modules/puppeteer/lib/DOMWorld.js:315:21)
    at Frame.waitForSelector (/opt/hostedtoolcache/Python/3.7.10/x64/share/jupyter/lab/browser_test/node_modules/puppeteer/lib/FrameManager.js:384:51)
    at Frame.<anonymous> (/opt/hostedtoolcache/Python/3.7.10/x64/share/jupyter/lab/browser_test/node_modules/puppeteer/lib/helper.js:117:27)
    at Page.waitForSelector (/opt/hostedtoolcache/Python/3.7.10/x64/share/jupyter/lab/browser_test/node_modules/puppeteer/lib/Page.js:867:33)
    at main (/opt/hostedtoolcache/Python/3.7.10/x64/share/jupyter/lab/browser_test/chrome-test.js:28:25)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[C 2021-03-20 19:03:53.308 ServerApp] Caught exception during the test:
Caught exception during the test:
[E 2021-03-20 19:03:53.308 ServerApp] ['node', 'chrome-test.js', 'file:///home/runner/.local/share/jupyter/runtime/jpserver-2551-open.html'] exited with 1
['node', 'chrome-test.js', 'file:///home/runner/.local/share/jupyter/runtime/jpserver-2551-open.html'] exited with 1
[I 2021-03-20 19:03:53.308 ServerApp] Test Complete
[C 2021-03-20 19:03:53.308 ServerApp] Exiting with 1 due to errors
Exiting with 1 due to errors
[I 2021-03-20 19:03:53.308 ServerApp] Stopping server...
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -46,3 +46,4 @@ jobs:
         jlpm run eslint:check
         python -m pip install .
         jupyter labextension list 2>&1 | grep -ie "jupyterlab-notifications.*OK"
+        python -m jupyterlab.browser_check --no-chrome-test

--- a/src/index.ts
+++ b/src/index.ts
@@ -51,7 +51,7 @@ const extension: JupyterFrontEndPlugin<void> = {
   optional: [ISettingRegistry],
   activate: async (app: JupyterFrontEnd, settingRegistry: ISettingRegistry) => {
     checkBrowserNotificationSettings();
-    let enabled = false;
+    let enabled = true;
     let minimumCellExecutionTime = 60;
     let reportCellExecutionTime = true;
     let reportCellNumber = true;

